src/tools/scale_tool.h

```diff
@@ -65,6 +69,7 @@ Q_OBJECT
 	MapCoordF scaling_center;
 	double reference_length = 0;
 	double scaling_factor   = 1;
+	bool using_selection_center = true;
```

Comment:
I spotted the slight different naming of `scaling_center` and `using_selection_center`. What I found now is that while the selection center is the default, the use may explicitly choose another scaling_center. From that point of view, renaming `scaling_center` is not an option, and `using_selection_center` is not 100% at target.

Reply:
Ack. Will give it another attempt.

Reply:
So I resorted to `using_scaling_center`. Looks reasonably consistent to me at the moment.